components/trello/cards-due.js

```diff
@@ -0,0 +1,67 @@
+const trello = require("https://github.com/PipedreamHQ/pipedream/components/trello/trello.app.js");
+
+module.exports = {
+  name: "Cards Due",
+  description:
+    "Emits an event at a specified time before a card is due.",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    trello,
+    boardId: { propDefinition: [trello, "boardId"] },
+    timeBefore: {
+      type: "integer",
+      label: "Time Before",
```

Comment:
Consider changing this name to something like "Time Before Due Time", which is a mouthful but slightly more clear.